web3/contracts/FincubeDAO.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.8;
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract FincubeDAO is Ownable {
+    event MemberRegistered(address indexed _newMember, string _memberURI);
```

Comment:
I think adding @notice and @param comments will be a good practice for Solidity Events.